etp-backend/src/test/clj/solita/etp/service/valvonta_kaytto/toimenpide_test.clj

```diff
@@ -60,3 +60,18 @@
     (t/is (true? (toimenpide/sakkopaatos-toimenpide? {:type-id 14}))))
   (t/testing "Käskypäätös / kuulemiskirje is not recognized as sakkopäätös-toimenpide?"
     (t/is (false? (toimenpide/sakkopaatos-toimenpide? {:type-id 7})))))
+
+(t/deftest sakkopaatos-tiedoksianto-toinen-postitus-test
+  (let [
```

Comment:
Itekin haluaisin formatoida näin, mutta kuulemma aloittavat ja lopettavat sulut kuuluu samalle riville sisällön kanssa

Reply:
https://guide.clojure.style/ tässä on kuvattu yleisesti hyväksytyt formatointikäytännöt, kannattaa tutustua. Myös Cursiven formatteri noudattaa (pääsääntöisesti) näitä sääntöjä. Harmillisesti se ei kuitenkaan ota kantaa rivinvaihtoihin.